packages/cli/src/commands/gsn-relayer-withdraw.ts

```diff
@@ -0,0 +1,44 @@
+import { CommandsLogic, WithdrawOptions } from '../CommandsLogic'
+import { gsnCommander, getKeystorePath, getServerConfig } from '../utils'
+import { createCommandsLogger } from '../CommandsWinstonLogger'
+import { KeyManager } from '@opengsn/relay/dist/KeyManager'
+import { toBN } from 'web3-utils'
+
+const commander = gsnCommander(['f', 'g'])
+  .option('-k, --keystore-path <keystorePath>', 'relay manager keystore directory', '/app/data/gsndata/manager/')
+  .option('-s, --server-config <serverConfig>', 'server config file', '/app/data/config/gsn-relay-config.json')
+  .option('-b, --broadcast', 'broadcast tx after logging it to console', false)
+  .requiredOption('-a, --amount <amount>', 'amount of funds to withdraw to owner address, in wei')
```

Comment:
for UI, please use eth, not wei (as we do in fund, stake)

Reply:
Already fixed